samples/nrf9160/mqtt_simple/prj.conf

```diff
@@ -0,0 +1,38 @@
+# General config
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# Trusted execution
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
```

Comment:
This is now redundant when building for nrf9160_pca10090_ns_.

Reply:
Looks that this still isn't addressed.